### ISSUE TITLE
graphics bug, full screen, after minimizing.

### ISSUE THREAD

**Issue by Grelko** [CleverRaven/Cataclysm-DDA#20351]
When using "full screen" mode, if I minimize the game "windows button, alt+tab etc" and re-open the game screen, I get something like this (depending what's on the screen at the time). 

If I'm on the main screen, before entering the game, the entire screen looks like the messed up area.

Can be fixed by doing any of these. Restarting the game, save/quitting, exiting the game to main screen, or opening then closing world map.

http://imgur.com/a/M1g9r

Nvidia GeForce GT610 (My GeForce 8400GS did the same thing when I was playing this a few years ago) 

It's probably just my computer, but this is the only game it happens on.

Win xp SP3
Tiles
Version 0.C-21270-gd34c7b510b

**Comment by kolsurma** [comment]
Not just your computer, I get oddities with the game quite often, especially if I've left it open for a long enough period of time.

Usually it appears as a totally black screen, or a screen with characters overlapping the sidebar.

Next time I see it I'll take a screenshot. Shouldn't take too long.

**Comment by Grelko** [comment]
Every once in a while, if I minimize the game and get on the internet, after a while, the game crashes. I'll have to save a report next time it does.

**Comment by kolsurma** [comment]
Win10

An example of one type of graphical glitching. Occurred after windows entered the lock screen.
The character quick stats and the minimap have both glitched. Possibly due to the display buffers not updating when the focus comes back to the app.
![image](https://cloud.githubusercontent.com/assets/10261427/23565057/bdf09f1c-0011-11e7-985c-4b5a4d46ed63.png)

**Comment by Leland** [comment]
Yeah, there's a few open issues with Windows' application switching. For two other ones, see #14023 and #6928
